app/src/main/java/de/westnordost/streetcomplete/quests/crossing/AddCrossing.kt

```diff
@@ -0,0 +1,186 @@
+package de.westnordost.streetcomplete.quests.crossing
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.elementfilter.toElementFilterExpression
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Node
+import de.westnordost.streetcomplete.data.osm.mapdata.Way
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmElementQuestType
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+import de.westnordost.streetcomplete.util.initialBearingTo
+import de.westnordost.streetcomplete.util.normalizeDegrees
+
+class AddCrossing : OsmElementQuestType<Boolean> {
+
+    private val roadsFilter by lazy { """
+        ways with
+          highway ~ primary|primary_link|secondary|secondary_link|tertiary|tertiary_link|unclassified|residential
+          and area != yes
+          and crossing != no
```

Comment:
`crossing=no` is used on 272 ways worldwide, I would not consider it as a good or common tagging

In such case missing `crossing=no` on nodes should be added

See https://taginfo.openstreetmap.org/tags/crossing=no#overview

Reply:
ok